crates/ctl/src/cli/host/mod.rs

```diff
@@ -49,6 +52,8 @@ impl HostCommands {
             HostCommands::Status(status) => status.run(client).await,
 
             HostCommands::IdmSnoop(snoop) => snoop.run(client, events).await,
+
+            HostCommands::HypervisorMessages(hvdmesg) => hvdmesg.run(client).await,
```

Comment:
Hmm, Maybe we should rename this command to `hv-console` and the file should be `hv_console.rs`

Just to shorten the command :)

Reply:
Done in 7e26f68